samples/net/throughput_server/src/common.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define MY_PORT 42042
+
+#define MAX_DBG_PRINT 64
+
+struct tp_stats {
+	struct {
+		u32_t prev_recv;
+		u32_t recv;
+		u32_t sent;
+		u32_t dropped;
+	} pkt;
```

Comment:
Perhaps, if the other part of struct is called "bytes", this should be called "pkts"?